src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionMetric.kt

```diff
@@ -85,6 +85,18 @@ data class ActionMetric @Deprecated("Use ActionMetric.Builder instead.") constru
         private var drilldown: RecordedPerformanceEntries? = null
         private var virtualUser: UUID = UUID.randomUUID()
 
+
+        constructor(actionMetric: ActionMetric) : this(
```

Comment:
If we [avoid `var`s](https://github.com/atlassian/jira-actions/pull/59#discussion_r393700108), we won't need to copy `ActionMetric`s.